<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5SB7TM2" failures="2" tests="2" name="Complete test run" time="46.966" errors="0" timestamp="2020-07-30T12:03:45 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="46.078">
    <failure type="org.openqa.selenium.WebDriverException" message="Timed out waiting 45 seconds for Firefox to start.
Build info: version: &amp;apos;4.0.0-alpha-3&amp;apos;, revision: &amp;apos;8c567de6dc&amp;apos;
System info: host: &amp;apos;DESKTOP-5SB7TM2&amp;apos;, ip: &amp;apos;192.168.133.1&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;10.0.2&amp;apos;
Driver info: driver.version: FirefoxDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting 45 seconds for Firefox to start.
Build info: version: '4.0.0-alpha-3', revision: '8c567de6dc'
System info: host: 'DESKTOP-5SB7TM2', ip: '192.168.133.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '10.0.2'
Driver info: driver.version: FirefoxDriver
at org.openqa.selenium.firefox.xpi.XpiDriverService.waitUntilAvailable(XpiDriverService.java:236)
at org.openqa.selenium.firefox.xpi.XpiDriverService.start(XpiDriverService.java:159)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:576)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:210)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:130)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:154)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:132)
at steps.Hook.InitializeTest(Hook.java:45)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://localhost:15175/hub/status] to be available after 45002 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:86)
at org.openqa.selenium.firefox.xpi.XpiDriverService.waitUntilAvailable(XpiDriverService.java:234)
at org.openqa.selenium.firefox.xpi.XpiDriverService.start(XpiDriverService.java:159)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:576)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:210)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:130)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:154)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:132)
at steps.Hook.InitializeTest(Hook.java:45)
at io.cucumber.java.Invoker.invoke(Invoker.java:27)
at io.cucumber.java.JavaHookDefinition.execute(JavaHookDefinition.java:61)
at io.cucumber.core.runner.CoreHookDefinition.execute(CoreHookDefinition.java:31)
at io.cucumber.core.runner.HookDefinitionMatch.runStep(HookDefinitionMatch.java:20)
at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:64)
at io.cucumber.core.runner.TestStep.run(TestStep.java:49)
at io.cucumber.core.runner.TestCase.run(TestCase.java:47)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:67)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:124)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1378)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:84)
... 55 more
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="46.108">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Base.ExtentReportUtil.ExtentReportScreenshot(ExtentReportUtil.java:39)
at runner.NGTestListener.onTestFailure(NGTestListener.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1378)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Complete test run -->
